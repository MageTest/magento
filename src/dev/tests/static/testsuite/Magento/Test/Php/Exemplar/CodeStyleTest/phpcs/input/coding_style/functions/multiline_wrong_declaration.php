<?php
/**
 * Function with lot of parameters.
 * With long description.
 *
 * @param string|null $someLongParam
 * @param bool $anotherLongParam
 * @param int $moreEvenLongerParamForAllThatRoutineStuff
 * @param float $andThereGoesOneParameter
 * @return string
 */
function thereGoesFunc($someLongParam, $anotherLongParam, $moreEvenLongerParamForAllThatRoutineStuff,
    $andThereGoesOneParameter
){
    if ($someLongParam === null) {
        $someLongParam = 'Stranger';
    }
    return 'Hello, ' . $someLongParam . '!';
}

/**
 * Function with lot of parameters.
 * With long description.
 *
 * @param string|null $someLongParam
 * @param bool $anotherLongParam
 * @param int $moreEvenLongerParamForAllThatRoutineStuff
 * @param float $andThereGoesOneParameter
 * @return string
 */
function thereGoesAnotherFunc($someLongParam, $anotherLongParam, $moreEvenLongerParamForAllThatRoutineStuff,
    $andThereGoesOneParameter
    ) {
    if ($someLongParam === null) {
        $someLongParam = 'Stranger';
    }
    return 'Hello, ' . $someLongParam . '!';
}

/**
 * Function with lot of parameters.
 * With long description.
 *
 * @param string|null $someLongParam
 * @param bool $anotherLongParam
 * @param int $moreEvenLongerParamForAllThatRoutineStuff
 * @param float $andThereGoesOneParameter
 * @return string
 */
function thereGoesThirdFunc($someLongParam, $anotherLongParam, $moreEvenLongerParamForAllThatRoutineStuff,
    $andThereGoesOneParameter
)
{
    if ($someLongParam === null) {
        $someLongParam = 'Stranger';
    }
    return 'Hello, ' . $someLongParam . '!';
}

/**
 * Function with lot of parameters.
 * With long description.
 *
 * @param string|null $someLongParam
 * @param bool $anotherLongParam
 * @param int $moreEvenLongerParamForAllThatRoutineStuff
 * @param float $andThereGoesOneParameter
 * @return string
 */
function thereGoesFourthFunc($someLongParam, $anotherLongParam, $moreEvenLongerParamForAllThatRoutineStuff,
    $andThereGoesOneParameter)
{
    if ($someLongParam === null) {
        $someLongParam = 'Stranger';
    }
    return 'Hello, ' . $someLongParam . '!';
}

/**
 * Function with lot of parameters.
 * With long description.
 *
 * @param string|null $someLongParam
 * @param bool $anotherLongParam
 * @param int $moreEvenLongerParamForAllThatRoutineStuff
 * @param float $andThereGoesOneParameter
 * @return string
 */
function thereGoesFifthFunc($someLongParam, $anotherLongParam, $moreEvenLongerParamForAllThatRoutineStuff,
    $andThereGoesOneParameter) {
    if ($someLongParam === null) {
        $someLongParam = 'Stranger';
    }
    return 'Hello, ' . $someLongParam . '!';
}
