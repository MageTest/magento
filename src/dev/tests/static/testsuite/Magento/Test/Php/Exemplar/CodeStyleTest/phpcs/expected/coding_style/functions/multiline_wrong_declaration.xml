<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="14"/>
    <error line="33"/>
    <error line="52"/>
    <error line="71"/>
    <error line="90"/>
</config>
